test/test-checks.el

```diff
@@ -15,16 +15,17 @@
 (setq checkdoc-arguments-in-order-flag nil)
 (setq checkdoc-verb-check-experimental-flag nil)
 
-(let ((files (directory-files default-directory t
-                              "\\`[^.].*\\.el\\'" t)))
+(when nil ;; See https://github.com/clojure-emacs/clj-refactor.el/issues/491
```

Comment:
I think you can also set some env var and just disable this on the CI. I think it's not a bad idea for people to be able to run this locally.